documentation/content/en/books/handbook/jails/_index.adoc

```diff
@@ -636,6 +636,13 @@ thinjail {
 }
 ....
 
+The name of a file in [.filename]#jail.conf.d# must be thinjail.conf. Otherwise you have to create also [.filename]#/etc/jail.conf#:
```

Comment:
You can add the reason here, such as the requirement for the file name to match the jail name, in this case, thinjail.conf.